### DISCUSSION TITLE
Adding MDC parameter as additional-field in quarkus json log

### DISCUSSION THREAD

**Discussion by rajagopal-mani** [quarkusio/quarkus#39139] [Q&A]
I am using io.quarkus:quarkus-logging-json in my quarkus app. The document says that to add additional field in json, I should use below property, 

`quarkus.log.console.json.additional-field."field-name".value=`

How to use MDC.get(paramName) as the value for this additional field ?

**Comment by matheuscirillo** [comment]
I don't think there's a way to do it, but would be great if the following work:
`quarkus.log.console.json.additional-field."test".value=%X{mdc-value}`
